client/src/Layout/Footer.tsx

```diff
@@ -0,0 +1,178 @@
+import { Box, Container, Grid2, Typography ,Divider} from "@mui/material"
+import React from "react"
+import Input from "../component/Input"
+import SendIcon from "@mui/icons-material/Send"
+import Qrimagefooter from "../image/Frame 719.png"
+import FacebookIcon from "@mui/icons-material/Facebook"
+import InstagramIcon from "@mui/icons-material/Instagram"
+import TwitterIcon from "@mui/icons-material/Twitter"
+import LinkedInIcon from "@mui/icons-material/LinkedIn"
+import { styled } from '@mui/system'
+
+const FooterBox = styled(Box)`
+  background-color: black;
+  padding: 30px 0px 30px 0px;
+
+`
+const FooterInput = styled(Box)`
+  border: 1px solid white;
+  width: 210px;
+  height: 46px;
+  margin-top: 16px;
+`
+
+const ConatinerFooter = styled(Container)(({ theme }) => ({
+  marginTop: "40px",
+  marginBottom: "30px"
+}))
+
+const StyledDivider = styled(Divider)(({ theme }) => ({
+  border: `1px solid #262525`
+}))
+const Footimg = styled("img")(() => ({
+
+  width: " 204px",
+  height: "85px"
+}))
+
+export default function Footer() {
+  return (
+    <>
+      <FooterBox>
+        <ConatinerFooter>
+          <Grid2 container columnSpacing={5}>
+            <Grid2 size={3}>
+              <Box>
+                <Typography variant="h4" sx={{ color: "white", mb: 2 }}>
```

Comment:
why using inline styling use to style components
 all inline style remove

Reply:
all the inline css is removed and changed  it into style component